sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -3410,4 +3410,22 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
       ctx.EXISTS != null,
       ctx.TEMPORARY != null)
   }
+
+  override def visitCommentNamespace(ctx: CommentNamespaceContext): LogicalPlan = withOrigin(ctx) {
+    val comment = ctx.commennt.getType match {
+      case SqlBaseParser.NULL => ""
```

Comment:
shall we distinguish between null and empty string? maybe the `comment` should be `Option[String]`

Reply:
null to delete the key? but reserved ones are tenured

Reply:
> null to delete the key

That's my expectation. What's the behavior of pgsql and presto? I feel it's a reasonable request to remove a comment.

Reply:
I have checked the Postgres source code `NULL` and empty string are the same, 
```c
/*
 * CreateComments --
 *
 * Create a comment for the specified object descriptor.  Inserts a new
 * pg_description tuple, or replaces an existing one with the same key.
 *
 * If the comment given is null or an empty string, instead delete any
 * existing comment for the specified key.
 */
```

Reply:
but null does mean to delete the comment, right? We don't need `Option[String]` because empty string also means to delete the comment.

Reply:
yes

Reply:
for anyone may concern about this, please refer to https://github.com/postgres/postgres/blob/master/src/backend/commands/comment.c#L133